Nbuild/resources/app-Ntools.json

```diff
@@ -3,7 +3,7 @@
   "NbuildAppList": [
     {
       "Name": "Ntools",
-      "Version": "1.3.7",
+      "Version": "1.3.0",
```

Comment:
Changed to a release version. This will change when another release is published.